modules/web/src/app/shared/components/labels/component.ts

```diff
@@ -46,6 +62,18 @@ export class LabelsComponent implements OnInit, OnChanges {
     return hiddenLabels;
   }
 
+  showAndHideAllLabels(): void {
+    this.showHiddenLabels = !this.showHiddenLabels;
+  }
+
+  private _checkLabelsHeight(): void {
+    const labelsElem = 30;
+    if (document.getElementById('km-labels').offsetHeight > labelsElem) {
```

Comment:
You should inject `private readonly _elRef:ElementRef` in `constructor` and then do this:
```suggestion
    if (this._elRef.nativeElement.offsetHeight > labelsElem) {
```

If this doesn't work then please use `@ViewChild` for getting reference of `div`.